MonkeyBot/Database/MonkeyDBContext.cs

```diff
@@ -83,8 +83,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             _ = modelBuilder.Entity<GameServer>().Property(x => x.GameServerType).IsRequired().HasConversion<string>();
             _ = modelBuilder.Entity<GameServer>().Property(x => x.ServerIP).IsRequired()
                 .HasConversion(
-                    x => JsonConvert.SerializeObject(x),
-                    x => JsonConvert.DeserializeObject<System.Net.IPEndPoint>(x));
+                    x => JsonSerializer.Serialize(x, null),
+                    x => JsonSerializer.Deserialize<System.Net.IPEndPoint>(x, null));
```

Comment:
This throws for some reason. I think it's safe to replace with `ToString()` and `System.Net.IPEndPoint.Parse()` anyway